pandas/core/reshape.py

```diff
@@ -1075,7 +1077,15 @@ def get_dummies(data, prefix=None, prefix_sep='_', dummy_na=False,
         If appending prefix, separator/delimiter to use. Or pass a
         list or dictionary as with `prefix.`
     dummy_na : bool, default False
-        Add a column to indicate NaNs, if False NaNs are ignored.
+        Add a column to indicate NaNs if True.
+    fill_value : scalar, default None
+        Value to fill NaNs with. The default of `None` will fill with
+        zeros. To do no filling of NaNs, specify `fill_value=np.nan`.
+        The default behavior of filling with zeros will be deprecated
+        in the future and using this default will not raise a
+        `DeprecationWarning`.
```

Comment:
@beckermr IIRC you asked about FutureWarning vs. DeprecationWarning. Python doesn't display DeprecationWarnings to the user by default, so we have FutureWarning which will be visible to the users.